src/Uno.UI/Extensions/UIViewExtensions.iOSmacOS.cs

```diff
@@ -532,22 +532,6 @@ public static nfloat GetNativeAlpha(this _View view)
 #endif
 		}
 
-		/// <summary>
-		/// Gets an identifier that can be used for logging
-		/// </summary>
-		public static string GetDebugIdentifier(this _View element)
```

Comment:
What's the reason to remove this? (Haven't used it, not particularly attached to it, just curious)

Reply:
Because I've added a new one which is multi targetted, this one would have conflict.